t/quicksearch.t

```diff
@@ -22,12 +22,11 @@ use ok 'Bugzilla::Search';
 use ok 'Bugzilla::Search::Quicksearch';
 
 my $CGI = mock 'Bugzilla::CGI' => (add_constructor => [fake_new => 'hash',]);
-Bugzilla->usage_mode(USAGE_MODE_MOJO);
 Bugzilla->request_cache->{cgi} = Bugzilla::CGI->fake_new();
 
 like(
   dies { quicksearch('') },
-  qr/buglist_parameters_required/,
+  qr/without any search terms/,
```

Comment:
> What does this do?

before the fix, the mojo error screen would should the template var name `buglist_parameters_required` instead of the proper more friendly error description. This fix now show the proper error description so the text that is being looked for in the test script had to be updated.

Reply:
That is correct.